### ISSUE TITLE
:loudspeaker: Flyway Community DB Support moving to https://github.com/flyway/flyway-community-db-support

### ISSUE THREAD

**Issue by Barry-RG** [flyway/flyway#3832]
As many of you are probably very aware, Flyway has an issue in regards to its lead time in merging PRs from the community. 

This is due to a hardcoded build mechanism that we use to deploy to flyway/flyway which has been there since Flyway's early releases and which we have not been able to move away from. Over the last couple of years the use of this system has become a hindrance and prevented us from interacting with our open source PRs in the way that we would wish. As a result, we are looking at ways to improve this in a method which does not significantly disrupt Flyway development.

The [flyway/flyway-comunity-db-support](https://github.com/flyway/flyway-community-db-support) repository is one of those ways. 
since database contributions are normally the most self contained. As a result, we have migrated the flyway-community-db-support module and its children to this repository from `flyway/flyway` to allow us to more quickly and directly interact with contributions and requests from the community without the barriers we have had before. 

We understand that doing this on a non-major release may be a surprise, but we felt getting a pipeline that allowed Flyway to work better as an open source project was more important.

However, this does mean that any currently open PRs for Database support will have to be migrated by their owners to [flyway/flyway-comunity-db-support](https://github.com/flyway/flyway-community-db-support). We have not wanted to make this announcement earlier so that migrated PRs did not seem to ignored once that reached their new destination.

**Comment by cowwoc** [comment]
I'm confused by this announcement.

Are you saying that you want the community to file all issues and PRs against https://github.com/flyway/flyway-community-db-support instead of this repository? And you will then push changes from https://github.com/flyway/flyway-community-db-support back into this repository?

**Comment by osvetlik** [comment]
> I'm confused by this announcement.
> 
> Are you saying that you want the community to file all issues and PRs against https://github.com/flyway/flyway-community-db-support instead of this repository? And you will then push changes from https://github.com/flyway/flyway-community-db-support back into this repository?

If I understand it correctly, nobody will, they build the packages directly from the other repo.

**Comment by cowwoc** [comment]
@osvetlik Are you saying that Redgate plans to abandon this repository altogether and use https://github.com/flyway/flyway-community-db-support from now on?

@Barry-RG Care to clarify?

**Comment by osvetlik** [comment]
@cowwoc No, I'm saying this repository is for Flyway/Redgate, that one is for community db support. Core database support (provided by Redgate) remains here, community-provided drivers are in the new repository.

**Comment by Barry-RG** [comment]
Thanks for your reply @osvetlik , that is correct.

@cowwoc To go into more detail, core development of Flyway which is provided by Redgate remain here. PRs for core features and bug fixed from the open source community can be contributed here. However, there is a process that goes on here which Redgate must fulfil before they get merged.
New database integrations which the open source community which to provide and support now live in their own [repository]([https://github.com/flyway/flyway-community-db-suppor](https://github.com/flyway/flyway-community-db-support)). This allows them to be in their own process which should speed up our ability to accept contributions. These are built and available via GitHub Packages and Maven Central. The builds here pull these as dependencies from those sources. 

Before these were moved, everything was part of this repository and had to go through the same lengthy process.
